portfolio/src/main/webapp/comments.html

```diff
@@ -6,14 +6,9 @@
         <link rel="stylesheet" href="style.css">
         <script src="script.js"></script>
       </head>
-<body onload="getMessages()">
+<body onload="generateNavbar() ; getMessages()">
```

Comment:
Can these both be called by a single method so that only one method has to be called here in onload?

Reply:
Sure i'll work on a init function

Reply:
It looks like this comment is not addressed yet. (Usually we mark a conversation resolved if it's addressed in a commit. In this case, we should keep it open)

Since it's a small change, I'll approve it and leave it to you to address in this PR or next.